joern-cli/frontends/javasrc2cpg/src/main/scala/io/joern/javasrc2cpg/passes/AstCreator.scala

```diff
@@ -726,10 +726,20 @@ class AstCreator(filename: String, javaParserAst: CompilationUnit, global: Globa
   private def astForFieldVariable(v: VariableDeclarator, fieldDeclaration: FieldDeclaration): Ast = {
     // TODO: Should be able to find expected type here
     val annotations = fieldDeclaration.getAnnotations
+
```

Comment:
This means that we're throwing that information away. Isn't it possible to just ignore the information downstream? This turns `List<Customer>` into `List`, right?

Reply:
Yes, makes sense, we should not loose this information